src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -11,6 +11,10 @@ $bcs-task-left-offset: 10 * $bcs-task-grid + 2px;
     &.bcs-is-pending {
         opacity: .4;
     }
+
+    .bdl-Media-body {
```

Comment:
you could also add a class to the component instead of adding overloads to the name here